tests/unit_tests/test_kernel_applier.py

```diff
@@ -0,0 +1,62 @@
+from SyntheticDataset.image_operations import Point
+import unittest
+from SyntheticDataset.image_operations import KernelApplier
+from PIL import Image
+
+class KernelApplierTestCase(unittest.TestCase):
+
+    def setUp(self):
+        #self.test_kernel1 = [[32,25,26], [30,20,26], [35,25,26]]
+        #self.test_kernel1 = [[0,-1,0], [-1,5,-1], [0,-1,0]]
+        self.test_kernel1 = [[0,0,0], [0,1,0], [0,0,0]]
+        self.kernelSum = KernelApplier.getKernelSum(self.test_kernel1)
```

Comment:
This variable should be lowercase

Reply:
Additionally, this line is a part of the ```test_get_kernel_sum``` test and so it should be put into that method.